models/order.js

```diff
@@ -0,0 +1,42 @@
+const { Model, DataTypes } = require('sequelize');
+const sequelize = require('../config/connection');
+
+class Order extends Model {}
+
+Order.init(
+  {
+
+    order_id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true

```

Comment:
do we need to have a default value ? 
defaultValue: 1,
https://sequelize.org/docs/v6/core-concepts/model-basics/